src/global.scss

```diff
@@ -1,4 +1,4 @@
-@import 'node_modules/govuk-frontend/govuk/all';
+@import "node_modules/govuk-frontend/govuk/all";
```

Comment:
let catch up on this file @rspasici. Seems your editor is inserting double quotes